_posts/2015-09-4-automated-testing-and-continuous-integration.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Automated Testing with PHPUnit and Continuous Integration"
+author: sachit
+modified: 2015-09-4
+tags: [ phpunit, unit tests, automated testing, continuous integration, shippable]
+---
+
+Well I'm going to be making a [presentation](http://sachit-singh.github.io/kswsbs/) today at the [14th PHP developers meetup](https://developers-nepal.github.io/php/meetups/05-Sep-2015/) on the title of this post. Today I will be talking about [Automated](https://en.wikipedia.org/wiki/Test_automation) [Unit Testing](https://en.wikipedia.org/wiki/Unit_testing) and [Continuous Integration](https://en.wikipedia.org/wiki/Continuous_integration). I am going to be explaining about why testing, in general is important, the features of unit tests and why [CI](https://en.wikipedia.org/wiki/Continuous_integration) is cool.
```

Comment:
`I'm going to present...`

remove `on the title of this post`  and `Today`

Link CI just once.